Pod/Classes/VOKCoreDataManager.m

```diff
@@ -296,6 +327,38 @@ - (NSArray *)importArray:(NSArray *)inputArray forClass:(Class)objectClass withC
     return [returnArray copy];
 }
 
+- (NSArray *)arrayToImportAfterImportingRepeatedUniqueKeys:(NSArray *)repeatedUniqueKeys
+                                   forForeignUniqueKeyPath:(NSString *)foreignUniqueKeyPath
+                                           usingInputArray:(NSArray *)inputArray
+                                                  forClass:(Class)objectClass
+                                               withContext:(NSManagedObjectContext *)contextOrNil
+{
+    NSMutableArray *uniquedInputArray = [NSMutableArray arrayWithArray:inputArray];
+    
+    NSMutableArray *itemsToRemove = [NSMutableArray array];
+    for (id repeatedKey in repeatedUniqueKeys) {
+        for (id item in inputArray) {
```

Comment:
Actually, could this be:
```objc
for (id item in inputArray) {
    if ([repeatedUniqueKeys containsObject:[item valueForKeyPath:foreignUniqueKeyPath]]) {
```
(including replacing the `if` statement right below here).